modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaOperationModelParser.java

```diff
@@ -406,6 +408,11 @@ public Optional<SinceMuleVersionModelProperty> getSinceMuleVersionModelProperty(
     return JavaExtensionModelParserUtils.getSinceMuleVersionModelProperty(operationElement);
   }
 
+  @Override
+  public Optional<String> getMinMuleVersion() {
+    return of(calculateFromMethod(operationElement));
```

Comment:
Checking the implementation, I think that this does not take into account if the configuration/extension is annotationed with the sdk-api @Operation (which may raise the minMuleVersion)